### ISSUE TITLE
Maintain Context When Reloading File

### ISSUE THREAD

**Issue by dshastry** [mjmlio/mjml-app#250]
I have an MJML style file called in via an mj-include like so

```
  <mj-head>
<mj-include path="../../Styles/master-style.mjml.css" />
</mj-head>
```

This works great, no issues here. Every time I make changes to the external file I do not see a change in the current file, which is understandable as I assume the app isn't watching all files only the open one. However, if I try to reload my current file so I can see my changes, by clicking View-> Reload or using the keyboard shortcut it refreshes the whole app and resets me back to the top of the directory structure. This is very frustrating when trying to work with multiple MJ includes.

I am currently circumventing this by just having a dummy text file in my active directory and tabbing between my actual file and this file just so I can refresh.

It would be very helpful to have a refresh function that can reload the current working file without reloading the whole app.

**Comment by meriadec** [comment]
Watching all files that are "mj-included" is a bit tricky, because it would require to pre-parse the file and maintain a dynamic list of watched files.

What about having a custom shortcut/button to reload the current opened file without reloading the whole app? Would it be an acceptable trade-off?

**Comment by meriadec** [comment]
Yeah, if it is only a reload of the current template _content_ (basically only a re-render with MJML), it will not move the folder tree in any way.

If someone want to work on that, a good start is to start by adding [keyboard shortcut](https://github.com/mjmlio/mjml-app/blob/master/src/menu.js#L81-L87) or button in the UI, then connect it to a MJML re-render. (I don't currently have time to work on it)

**Comment by iRyusa** [comment]
This behavior is done in MJML-cli side, may be we can do something similar on the APP 🤔
